src/lf_apply.rs

```diff
@@ -687,10 +704,10 @@ pub(crate) unsafe fn rav1d_loopfilter_sbrow_cols<BD: BitDepth>(
         }
     }
     let mut ptr: *mut BD::Pixel;
-    let level_ptr = &(*f).lf.level[((*f).b4_stride * sby as isize * sbsz as isize) as usize..];
+    let mut level_ptr = &(*f).lf.level[((*f).b4_stride * sby as isize * sbsz as isize) as usize..];
     ptr = p[0];
     have_left = 0 as c_int;
-    for (x, level_ptr) in (0..(*f).sb128w).zip(level_ptr.chunks(32)) {
+    for x in 0..(*f).sb128w {
```

Comment:
I think this relates to the fact that you may need more than 32 elements from `level_ptr` down the road.

Reply:
that makes sense (the current test cases don't as far as I know, but in general that might be the case)

Reply:
I had noticed some slicing out of bounds errors, so I removed that extra limit of `32`/`32 >> ss_ver` that excluded padding.  It should also be slightly faster I think, since it's doing one less bounds check.